### ISSUE TITLE
Upcoming shortcode - shows only 4 events

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#452]
**Describe the bug**

I added the Upcoming shortcode. There are 6 events and it shows only 4.

**To Reproduce**
1. Create an upcoming shortcode.
2. Add the shortcode to a page.
3. Notice only 4 events are seen. 

**Expected behavior**

I expected all the upcoming events to be seen. Or alteast having a method to add how many upcoming events should be seen.

**Comment by paaljoachim** [comment]
Adding a few more steps.

1. Create an event with two additional copies.
Result:

![Screenshot 2022-02-14 at 17 07 52](https://user-images.githubusercontent.com/5323259/153900973-23319088-b2ab-4492-a0d0-d31d2e5332bb.jpg)

Bonus: See if you can select -- as location. I was not able to, so it went to the automatic location. As I had deleted the default location it used the location I had created earlier.

2. Hover the quick edit and select copy.
![Screenshot 2022-02-14 at 17 08 32](https://user-images.githubusercontent.com/5323259/153901076-657ee1d6-ea50-4137-93ca-f7abebf7db81.jpg)

Result (the event group is duplicated showing three duplicate events). After having adjusted the new events these now look like this:
![Screenshot 2022-02-14 at 17 07 59](https://user-images.githubusercontent.com/5323259/153901114-185a169c-d76a-4884-bff6-aed4abdba261.jpg)

3. Add an upcoming shortcode and view the frontend:
It might look something like this:
It does not show the added copies. Only the first event.

![Screenshot 2022-02-14 at 17 10 46](https://user-images.githubusercontent.com/5323259/153901594-719b5358-6746-4341-b409-22a007a7524b.jpg)

**Comment by joedolson** [comment]
Is one of these events the grouped event you checked as a multi-day event? If that's so, then it's being collapsed into a single entry.

**Comment by joedolson** [comment]
Yes. The 'before' parameter sets a count on past events, and the 'after' parameter sets a count on future events. 'show_today' sets whether or not today's events are included.

**Comment by paaljoachim** [comment]
Wha?......:)
I see the "Events/Days Before Current Day" and "Events/Days After Current Day"
I find the wording for both confusing. 

I am using the shortcode:
[my_calendar_upcoming]

I am now testing:
[my_calendar_upcoming before="10" after="10" type="event" order="asc" template="abcd206b575b82d604522ca7bf01021f"]

It looks like it helped....
https://fragmentsofawalk.com/norske-okosamfunns-forening/aktiviteter/

**Comment by joedolson** [comment]
Yes, they can be confusing. It's because the fields representing different things depending on other settings. They're events if the type=events, but days if type=days. E.g., showing the next 10 events vs. showing events in the next 10 days. It's hard to communicate clearly.

**Comment by joedolson** [comment]
This is mostly related to legacy template editing, ultimately, and is no longer relevant since that's now a legacy system.
